src/HasNumericId.java

```diff
@@ -0,0 +1,4 @@
+public interface HasNumericId {
```

Comment:
could be just abstract class. In this case you don't need to overide getter and setter, and add id field in each entity

Reply:
I was thinking this way: we already have three entities: Ticket, Client, and Admin. Users could also have their numeric ID. However, they wouldn't have the chance to do so if HasNumericId was an abstract class.

Reply:
Up to you) its just practical example)